api/prisma/migrations/16_add_duplicates_view/migration.sql

```diff
@@ -0,0 +1,43 @@
+CREATE VIEW "application_flagged_set_possibilities" AS (
+    SELECT
+        LOWER(a.first_name) || '-' || LOWER(a.last_name) || '-' || a.birth_month || '-' || a.birth_day || '-' || a.birth_year as "key",
+        app.listing_id,
+        app.id as "application_id",
+        'nameAndDOB' as "type"
+    from
+        applicant a,
+        applications app
+    where
+        a.id = app.applicant_id
+)
+UNION
+(
+    SELECT
+        a.email_address as "key",
+        app.listing_id,
+        app.id as "application_id",
+        'email' as "type"
+    from
+        applications app,
+        applicant a
+    where
+        a.id = app.applicant_id
+        and a.email_address is not null
+)
+UNION
+(
+    SELECT
+        LOWER(hm.first_name) || '-' || LOWER(hm.last_name) || '-' || hm.birth_month || '-' || hm.birth_day || '-' || hm.birth_year as "key",
+        app.listing_id,
+        app.id as "application_id",
+        'nameAndDOB' as "type"
+    from
+        applications app,
+        household_member hm
+    where
+        hm.application_id = app.id
+);
+
+ALTER TYPE "rule_enum"
+ADD
+    VALUE 'emailAndNameAndDOB';
```

Comment:
I don't remember what version of postgres we run in prod but may want to peep: https://stackoverflow.com/questions/1771543/adding-a-new-value-to-an-existing-enum-type

and double check the versioning to make sure this will work in our various production envs

Reply:
I checked and all of our production environments support this.